stories/form/form-unit-number-input/props-table.tsx

```diff
@@ -43,6 +43,11 @@ const DATA: ApiTableSectionProps[] = [
                 description: "The unique identifier of the component",
                 propTypes: ["string"],
             },
+            {
+                name: "placeholder",
+                description: "The placeholder of the component",
```

Comment:
should we keep the original note on the placeholder format here? passing in any other string that doesn't match `<floor>-<unit>` format may have unexpected results

Reply:
It's meant to be free and easy. But i'll add a recommendation instead